index.html

```diff
@@ -253,57 +256,59 @@ <h4>Skills</h4>
         </div>
-       
+    
+        <footer>
+            <div class="footer">
+                <p class="get-in-touch">
+                    I'm always interested in hearing about new projects, 
+                    so if you'd like to chat please get in touch.
+                </p>
         
-        <img src="images/contact-form-comp.svg" class="contact-img" alt="contact-form-img">
-
-        <div class="line"></div>
-
-        <ul class="handle-icon set-icons">
-            <li><a href=""><img class="svg" src="images/github.svg" alt="Github logo"></a></li>
-            <li><a href=""><img class="svg" src="images/Linkedin-icon.svg" alt="LinkedIn logo"></a></li>
-            <li><a href=""><img class="svg" src="images/angellist.svg" alt="angellist logo"></a></li>
-            <li><a href=""><img class="svg" src="images/twitter.svg" alt="twitter logo"></a></li>
-            <li><a href=""><img class="svg" src="images/medium.svg" alt="Medium logo"></a></li>
+                <form action="https://formspree.io/f/mpzbgqgr" method="POST">
+                    <ul>
+                        <li>
+                            <input type="text" id="f-name" name="text" maxlength="30" placeholder="Full Name" class="full-name" required>
+                            <label for="f-name"></label>
+                        </li>
+                        <li>
+                            <input type="email" id="email" name="email" placeholder="Email address" class="email" required>
+                            <label for="email"></label>
+                        </li>
+                        <li>
+                            <textarea name="comment" maxlength="500" placeholder="Enter text here" class="textarea" required></textarea>
+                        </li>
+                        <li>
+                            <button class="button btns form-btn " type="submit">Get in touch</button>
+                        </li>
+                    </ul>
+    
+                </form>
+            </div>
+           
             
-        </ul>
-
-        <p class="gmail">ehmaessien@gmail.com</p>
-    </footer>
+            <img src="images/contact-form-comp.svg" class="contact-img" alt="contact-form-img">
+    
+            <div class="line"></div>
+    
+            <ul class="handle-icon set-icons">
+                <li><a href=""><img class="svg" src="images/github.svg" alt="Github logo"></a></li>
+                <li><a href=""><img class="svg" src="images/Linkedin-icon.svg" alt="LinkedIn logo"></a></li>
+                <li><a href=""><img class="svg" src="images/angellist.svg" alt="angellist logo"></a></li>
+                <li><a href=""><img class="svg" src="images/twitter.svg" alt="twitter logo"></a></li>
+                <li><a href=""><img class="svg" src="images/medium.svg" alt="Medium logo"></a></li>
+                
+            </ul>
+    
+            <p class="gmail">ehmaessien@gmail.com</p>
+        </footer>
```

Comment:
### Figma Template
![179849920-0b0f0749-b338-415f-83d7-4befbc01091d (copy)](https://user-images.githubusercontent.com/93039359/179881185-e45c4624-0815-4120-96a5-9687d34ed262.png)

**[Optional]**
- [ ] It would be nice if you add the black bar at the end of your menu to follow the Figma template :+1: